shotover/src/runner.rs

```diff
@@ -245,6 +241,73 @@ impl Shotover {
     }
 }
 
+pub struct Server {
```

Comment:
What is the purpose of this Server struct?
If the purpose is to reduce the number of arguments to `run`, can we just make `run` a member of `Shotover` instead of introducing a new type that we only use once?